src/import/ImportFFmpeg.cpp

```diff
@@ -522,12 +522,11 @@ ProgressResult FFmpegImportFileHandle::Import(WaveTrackFactory *trackFactory,
    }
    // This is the heart of the importing process
    // The result of Import() to be returned. It will be something other than zero if user canceled or some error appears.
-   auto res = ProgressResult::Success;
-
+   auto hasError {false};
```

Comment:
This is an unusual way to write it.

Funny C++ fact:  if you change it to
`auto hasError = {false};`

then instead they type is deduced as `std::initializer_list<bool>` and compilation fails because that is not convertible too bool.